daemon/cluster/executor/container/attachment.go

```diff
@@ -19,8 +20,8 @@ type networkAttacherController struct {
 	closed  chan struct{}
 }
 
-func newNetworkAttacherController(b executorpkg.Backend, task *api.Task) (*networkAttacherController, error) {
-	adapter, err := newContainerAdapter(b, task)
+func newNetworkAttacherController(b executorpkg.Backend, task *api.Task, secrets exec.SecretGetter) (*networkAttacherController, error) {
+	adapter, err := newContainerAdapter(b, task, secrets)
```

Comment:
actually i was wrong.  `newContainerAdapter` needs the SecretGetter and the networkAttacherController instantiates a ContainerAdapter so it needs passed.  perhaps it can be nil; i'm not sure how the network attacher works.